package.json

```diff
@@ -26,6 +26,14 @@
         "types": "./dist/esm/syntax/index.d.ts",
         "default": "./dist/esm/syntax/index.js"
       }
+    },
+    "./types": {
+      "require": {
+        "types": "./dist/cjs/types.cts"
+      },
+      "import": {
+        "types": "./dist/esm/types.d.ts"
+      }
     }
```

Comment:
I think it's a good idea to export the `types` declaration in the package as it's done in `@eslint/json`: https://github.com/eslint/json/blob/json-v0.12.0/package.json#L20-L27